api/report_server.thrift

```diff
@@ -170,18 +177,18 @@ service codeCheckerDBAccess {
                        1: i64 reportId)
                        throws (1: shared.RequestFailed requestError),
 
-  // get the results for one runId
+  // get the results for some runIds
   ReportDataList getRunResults(
-                               1: i64 runId,
+                               1: list<i64> runIds,
```

Comment:
What is the point of optional list? Is there a semantic difference between not filled and empty list? Maybe we should keep all of the fields non-optional and use empty list to represent a missing filter?